schema/draft/schema.ts

```diff
@@ -26,34 +26,47 @@ export type ProgressToken = string | number;
  */
 export type Cursor = string;
 
-/** @internal */
-export interface Request {
-  method: string;
-  params?: {
+/**
+ * Common params for any request.
+ */
+export interface RequestParams {
+  /**
+   * See [General fields: `_meta`](/specification/draft/basic/index#meta) for notes on `_meta` usage.
+   */
+  _meta?: {
     /**
-     * See [General fields: `_meta`](/specification/draft/basic/index#meta) for notes on `_meta` usage.
+     * If specified, the caller is requesting out-of-band progress notifications for this request (as represented by notifications/progress). The value of this parameter is an opaque token that will be attached to any subsequent notifications. The receiver is not obligated to provide these notifications.
      */
-    _meta?: {
-      /**
-       * If specified, the caller is requesting out-of-band progress notifications for this request (as represented by notifications/progress). The value of this parameter is an opaque token that will be attached to any subsequent notifications. The receiver is not obligated to provide these notifications.
-       */
-      progressToken?: ProgressToken;
-      [key: string]: unknown;
-    };
+    progressToken?: ProgressToken;
     [key: string]: unknown;
   };
 }
 
+/** @internal */
+export interface Request {
+  method: string;
+  /**
+   * Intended to enable unofficial extensions of `RequestParams` without negatively impacting how a `Request` is validated by intermediary code
+   */
```

Comment:
I think we could make this a regular comment instead of a JSDoc comment:

```suggestion
  // Allow unofficial extensions of `Request.params` without impacting `RequestParams`.
```